src/k8s/pkg/utils/mapstructure.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"reflect"
+	"strings"
+	"unicode"
+
+	"github.com/mitchellh/mapstructure"
+	"gopkg.in/yaml.v2"
+)
+
+// YAMLToStringSliceHookFunc returns a mapstructure.DecodeHookFunc that converts string to []string by parsing YAML.
+func YAMLToStringSliceHookFunc(f reflect.Kind, t reflect.Kind, data interface{}) (interface{}, error) {
```

Comment:
nit: `any` instead of `interface{}`

Reply:
going by https://github.com/mitchellh/mapstructure/blob/main/mapstructure.go#L185-L197